az-digital/az_quickstart
# Closes #1550 Initial Intelligence Bank integration.

## Description
Adds the ability to download images from the DAM and manage as either remote media or local media.

## Related issues
Closes #1550 

Spike: https://github.com/az-digital/az_quickstart/issues/1313
EPIC: https://github.com/az-digital/az_quickstart/issues/1551
MVP Issue: https://github.com/az-digital/az_quickstart/issues/1550
MVP PR https://github.com/az-digital/az_quickstart/pull/1592
Follow-up issue: https://github.com/az-digital/az_quickstart/issues/1686
Follow-up issue: https://github.com/az-digital/az_quickstart/issues/1689
Follow-up issue: https://github.com/az-digital/az_quickstart/issues/1688



## How to test
Enable az_dam module.
Go to /admin/content/media
Click on  [+ Add DAM assets]
In the url field click the custom url checkbox
In the url field add assets.arizona.edu
click sign in 
Click sign in with net id
Close once you are signed in, go back to the Add DAM assets page and click continue, or sign in and continue
Now you should be able to find an image and Download it.



## Types of changes
New DAM integration feature
### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
